homeassistant/components/sensor/ebusd.py

```diff
@@ -0,0 +1,188 @@
+"""
+Support for Ebusd daemon for communication with eBUS heating systems.
+For more details about ebusd, please refer to the documentation at
+https://github.com/john30/ebusd
+"""
+
+from datetime import timedelta
+from datetime import datetime
+import logging
+import subprocess
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_MONITORED_VARIABLES, STATE_ON, STATE_OFF, STATE_UNKNOWN)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'ebusd'
+CONF_CIRCUIT = 'circuit'
+CONF_TIME = 0
+
+BASE_COMMAND = 'ebusctl read -m {2} -c {0} {1}'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=15)
+SCAN_INTERVAL = timedelta(seconds=30)
```

Comment:
Can be removed as you already attached a throttle to the update of the data object.